scripts/setup-centos.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+TARGET=$1
+
+# TODO: Assert TARGET is one of "el6" and "el7"
+
+echo "[Install] dependencies"
+sudo yum update
+
+sudo yum install -y linux-image-extra-$(uname -r)
+sudo yum install -y git curl wget
+
+echo "[Install] docker-compose"
+if [ "$TARGET" = "el6" ]; then
+  # el6 does not work with docker-compose 1.4+
```

Comment:
I'm OK with dropping EL6 for Docker here. Am thinking about dropping EL6 altogether once it goes End of Phase 2 in May this year. 

Should be a very small subset of people that want Docker + EL6 together. Most of those folks should be Ubuntu or EL7 users.